_config.yml

```diff
@@ -150,3 +147,12 @@ exclude:
   - package-lock.json
   - README.md
   - src
+  - CONTRIBUTING.md
+  - LICENSE.md
+  - assets
+
+assets:
+  sources:
+    - node_modules/uswds/dist/img
+    - node_modules/uswds/dist/js
+    - node_modules/uswds/dist/scss
```

Comment:
These are necessary so `jekyll-assets` knows where to look to find images, js, and scss when referenced from code in this template and added by the user.